docs/source/specification.md

```diff
@@ -22,102 +27,147 @@ For example:
+:::
 
 ### Datatype
-Datatypes are *required* to have one of the following names `ephys, behav, funcimg, histology`.
-If collected, `ephys, behav, funcimg` are *required* to be placed at the session level. 
-If collected, `histology` is *required* to be at subject level. 
 
-### Example
-A real project folder may look like: 
+The following datatypes are supported:
+
+* `ephys`: electrophysiology data (e.g. Neuropixel probes, tetrodes, etc.)
+* `behav`: behavioral data (video and audio files, response logs, etc.)
+* `funcimg`: functional imaging data (e.g. calcium imaging, voltage imaging, etc.)
+* `histology`: anatomical data (e.g. serial-2-photon images, histology slides, etc.)
+
+:::{note}
+Unlike the first three `datatypes` that belong at the `session` level, `histology` belongs at the `subject` level.
+:::
+
+### Example project folder
+A real project folder may look like:
+
 ```
 └── project/
     ├── rawdata/
-    │   └── sub-001/
-    │       ├── ses-001_id-5645332/
+    │   └── sub-001_id-5645332/
+    │       ├── ses-01_date-20230310/
     │       │   ├── ephys/
-    │       │   │   ├── sub-001_ses-001_recording.bin
-    │       │   │   └── sub-001_ses-001_probe.imec0
+    │       │   │   ├── sub-001_ses-01_recording.bin
+    │       │   │   └── sub-001_ses-01_probe.imec0
     │       │   └── behav/
-    │       │       ├── sub-001_ses-001_camera_1.wav 
-    │       │       └── sub-001_ses-001_responses.csv 
+    │       │       ├── sub-001_ses-01_camera1.wav 
+    │       │       └── sub-001_ses-01_responses.csv 
     │       └── histology/
     │           └── brain_image.tiff
     └── derivatives/
-        └── sub-001/
-            ├── ses-001_id-5645332/
+        └── sub-001_id-5645332/
+            ├── ses-01_date-20230310/
             │   ├── ephys/
-            │   │   └── sub-001_ses-001_spike_sorted_data.mat
+            │   │   └── sub-001_ses-01_spike-sorted-data.npy
             │   └── behav/
-            │       └── sub-001_ses-001_tracking_results.csv
+            │       └── sub-001_ses-01_tracking-results.csv
             └── histology/
-                └── sub-001_ses-001_cell_counts.csv
+                └── sub-001_ses-01_cell-counts.csv
 ```
 
 ## File Naming Conventions
 
-File names are *recommended* to be formatted as set of key-value pairs ending with an *optional* suffix and *required* file extension (e.g. `sub-001_ses-001_date-20220516_retinotopy.ap.bin`). 
+`SWC-Bluprint` does not impose any absolute requirements on file names. That said, below we provide some ***recommended*** and ***optional*** guidelines for file names, based on the [BIDS specification](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filenames).
+
+:::{admonition} What makes a good file name?
+:class: tip
+* be nice to humans (readable and descriptive)
+* be nice to computers (parseable and consistent)
+* use alphanumeric characters `Aa-Zz, 0-9`, dashes `-`, underscores `_`
```

Comment:
could mention key-value pair arrangement is  preferred (maybe in the nice to computer line?)

Reply:
I don't want to blow up this section with too many examples (want to keep the "tip" very general). Anyhow, all of our recommendations in the below bullet points are essentially about the key-value pair convention.